packages/unchained-client/src/evm/parser/utils.ts

```diff
@@ -1,11 +1,9 @@
-import { EthereumTx } from '../../generated/ethereum'
-
 export const getSigHash = (inputData: string | undefined): string | undefined => {
   if (!inputData) return
   const length = inputData.startsWith('0x') ? 10 : 8
   return inputData.slice(0, length)
 }
 
-export const txInteractsWithContract = (tx: EthereumTx, contract: string) => {
+export const txInteractsWithContract = (tx: { to: string }, contract: string) => {
```

Comment:
```suggestion
export const txInteractsWithContract = (tx: Pick<EthereumTx, 'to'>, contract: string) => {
```

Reply:
The reason I just defined the object inline was to avoid using either the `EthereumTx` or `AvalancheTx` type within the generic evm utils, and considering how simple the logic is I didn't see too much harm. An alternate and probably useful update, would be exposing the base common api types as well so we could just use the `BaseTx` type here which all coinstack txs must extend from. Seemed overkill for this PR however